### PR TITLE
Release HA blog post

### PR DESCRIPTION
**Pull request by MathiasRenner**
Still work in progress, but open for comments.

### CODE REVIEW

**Comment by MathiasRenner** [comment]
Ok, the post is fine with me,  but please don't merge yet. I don't have the permission for the theme image yet (from kubecloud.io).

**Comment by MathiasRenner** [comment]
Update: I have the permission of kubecloud.io .

File: content/post/high-availability-with-docker.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+These two news combined beg for testing the new capabilities in reality.
+
+![SwarmClusterHA](/images/high-availability-testing/high-availability-docker-swarm.png)
+
+
+<!--more-->
+
+The "old" Docker Swarm already offered some high availability features, but the new called "Swarm Mode" goes far beyond: To me, the most important capabilities to mention are: built-in security, rolling updates and - maybe one of the most important benefits - super user-friendliness. Production ready clustering based on powerful container technology has probably never been easier before.
+
+Also, the release of Docker 1.12 comes with official support for the ARM architecture. Running Docker on ARM has been possible for long time, but now, users can install it the same way on their Raspberry Pis as they do it on any other architecture: Using Docker's official repositories. If you are one of our regular readers, you know that our Team at Hypriot intensively collaborated with Docker to make this official support possible. So we are also very happy that this huge milestone that many people asked for is eventually checked!
```
Comment (Govinda-Fichtner) [inline content/post/high-availability-with-docker.md:20]: Team -> team

File: content/post/high-availability-with-docker.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+In its release in June this year, Docker announced two exciting news about the Docker Engine: First, the Engine 1.12 comes with built-in high availability features, called "Docker Swarm Mode". And second, Docker started providing official support for the ARM architecture.
+
+These two news combined beg for testing the new capabilities in reality.
+
+![SwarmClusterHA](/images/high-availability-testing/high-availability-docker-swarm.png)
+
+
+<!--more-->
+
+The "old" Docker Swarm already offered some high availability features, but the new called "Swarm Mode" goes far beyond: To me, the most important capabilities to mention are: built-in security, rolling updates and - maybe one of the most important benefits - super user-friendliness. Production ready clustering based on powerful container technology has probably never been easier before.
```
Comment (Govinda-Fichtner) [inline content/post/high-availability-with-docker.md:18]: Production ready clustering based on powerful container technology has probably never been easier before. -> Clustering based on powerful container technology has probably never been easier before.

@MathiasRenner Do we have enough experience yet to say the above?

File: content/post/high-availability-with-docker.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+![Terminator](/images/high-availability-testing/terminal.png)
+
+That's all to do before getting our hands on Docker itself!
+
+
+Setup Docker Swarm on the cluster
+--------------------------------
+
+On latest HypriotOS, you'll have the latest Docker installed, so we can instantly start playing.
```
Comment (Govinda-Fichtner) [inline content/post/high-availability-with-docker.md:52]: Two times latest might not be the latest in eloquence... :)

File: content/post/high-availability-with-docker.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+I documented one of the test runs in the following screencast.
+
+<div align="center">
+<iframe src="https://player.vimeo.com/video/185361173" width="640" height="360" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
+<p><a href="https://vimeo.com/185361173">Testing High Availability Docker Swarm Mode</a> from <a href="https://vimeo.com/user54109827">Mathias Renner</a> on <a href="https://vimeo.com">Vimeo</a>.</p>
+</div>
+
+Results
+------------
+As shown in the screencast, Docker is able to recover from failure of the ethernet interface. After testing the other use cases later on, I can confirm that Docker recovers flawlessly from a reboot and crash as well. This holds for the slave and the master node, which is remarkable!
```
Comment (Govinda-Fichtner) [inline content/post/high-availability-with-docker.md:101]: Why is it remarkable?

File: content/post/high-availability-with-docker.md [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+
+
+More to come soon
+------------------
+This post is only a small chunk of the data I gathered during the tests. The next option to get the details is during my talk at the [HighLoad++ Conference](http://highload.co/) in Moskow, Russia. I'd be happy if you can make it there!
+
+Also, this is not the end of the story,kontakt@pringuin.de of course. Thorough testing requires also measuring incoming requests from an external load tester to the cluster while a failure occurs. After some research, I have not found any evidence that someone has ever performed that tests (please correct me if I'm wrong!).
```
Comment (Govinda-Fichtner) [inline content/post/high-availability-with-docker.md:108]: This sentence is all wrong: Also, this is not the end of the story,kontakt@pringuin.de of course.

**Review by Govinda-Fichtner — COMMENTED** [review]
@MathiasRenner Please have look at my comments....

File: content/post/high-availability-with-docker.md (hunk 2 above)
Comment (MathiasRenner) [reply content/post/high-availability-with-docker.md:18]: @Govinda-Fichtner Yes, but I am happy to see counterexamples.

File: content/post/high-availability-with-docker.md (hunk 1 above)
Comment (MathiasRenner) [reply content/post/high-availability-with-docker.md:20]: Thanks! Preparing commit to fix it.

File: content/post/high-availability-with-docker.md (hunk 3 above)
Comment (MathiasRenner) [reply content/post/high-availability-with-docker.md:52]: Agree :) I'll just delete the first "latest". Will be fixed with next commit.

File: content/post/high-availability-with-docker.md (hunk 5 above)
Comment (MathiasRenner) [reply content/post/high-availability-with-docker.md:108]: Outdated. I fixed this right after your first hint about this issue.

File: content/post/high-availability-with-docker.md (hunk 4 above)
Comment (MathiasRenner) [reply content/post/high-availability-with-docker.md:101]: I added some some more description here. Updated by a commit coming in some seconds...

**Pull request merged**